WebWasted/Controllers/FoodsController.cs

```diff
@@ -44,19 +44,33 @@ public IEnumerable<Food> Get()
             {
                 return DatabaseHandler.Instance.dc.Foods.ToList();
             }
-            
+
         }
-        
+        [HttpGet("{searchString}")]
+        public IEnumerable<Food> Get(string searchString)
+        {
+            lock (DatabaseHandler.Instance.dc)
+            {
+                var query = from food in DatabaseHandler.Instance.dc.Foods select food;
+                if (!string.IsNullOrEmpty(searchString))
+                {
+                    query = query.Where(offer => offer.Name.Contains(searchString) || offer.Description.Contains(searchString));
+                }
+                return query.ToList();
+            }
+
+        }
+        /*
         [HttpGet("{id}")]      //e.g. https://localhost:5000/api/foods/3
```

Comment:
Will we reuse this code? As I understand, the commented code is not neccessary, so we can just remove it

Reply:
Yep, I commented this fragment, because we never use it. I'll delete it